packages/esm-patient-list-app/src/api/api-remote.ts

```diff
@@ -30,11 +31,28 @@ async function postData(url: string, data = {}, ac = new AbortController()) {
   return response.data;
 }
 
+async function deleteData(url: string, data = {}, ac = new AbortController()) {
+  const response = await openmrsFetch(url, {
+    signal: ac.signal,
+    method: 'DELETE',
+    mode: 'cors',
+    cache: 'no-cache',
+    credentials: 'same-origin',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    redirect: 'follow',
+    referrerPolicy: 'no-referrer',
```

Comment:
Just curious, doesn't _necessarily_ have to be changed from my side: Are these headers actually required for the request to succeed or can they be omitted?